Overview.bs

```diff
@@ -368,8 +370,23 @@ not by wall-clock time, but by the progress of scrolling in a [=scroll container
 <div class="constructors">
 
 :   <dfn constructor for=ScrollTimeline lt="ScrollTimeline(options)">ScrollTimeline(options)</dfn>
-::  Creates a new {{ScrollTimeline}} object. The attributes of the {{ScrollTimeline}} are 
-    initialized with the respective members of the {{options}} dictionary.
+::  Creates a new {{ScrollTimeline}} object using the following procedure:
+
+    1. Let <var>timeline</var> be a new {{ScrollTimeline}} object.
+
+    1. Let <var>source</var> be the result corresponding to the first matching condition from below.
+
+        : If the <var>scrollSource</var> value of <var>options</var> is non-null,
+        :: Let <var>source</var> be <var>scrollSource</var>
+
+        : Otherwise (<var>scrollSource</var> is null):
+        :: Let <var>source</var> be the {{scrollingElement}} of the {{Document}} <a lt="document associated with a window">associated</a> with the {{Window}} that is the <a>current global object</a>.
```

Comment:
This looks good to me although I notice @bzbarsky's comment includes the step, "assert that it's a Window". I imagine that if we have `[Exposed=Window]` on this interface we are ok?

Reply:
Added Exposed=Window